### ISSUE TITLE
Error When Installing pyserial library in anaconda

### ISSUE THREAD

**Issue by gthiruna** [conda/conda#2947]
When I tried installing the pyserial library in anaconda, I am getting an unexpected error and I have attached error below.
Please help me to install pyserial library in anaconda

Fetching packages ...
conda-env-2.5. 100% |###############################| Time: 0:00:01  50.39 kB/s
pyserial-2.7-p 100% |###############################| Time: 0:00:01  67.67 kB/s
conda-4.1.5-py 100% |###############################| Time: 0:00:02 124.11 kB/s
Extracting packages ...
An unexpected error has occurred, please consider sending the            |   0%
following traceback to the conda GitHub issue tracker at:

```
https://github.com/conda/conda/issues
```

Include the output of the command 'conda info' in your report.

Traceback (most recent call last):
  File "C:\Users\gthir\Anaconda2\Scripts\conda-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\cli\main.py", line 120, in main
    exit_code = args_func(args, p)
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\cli\main.py", line 130, in args_func
    exit_code = args.func(args, p)
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\cli\main_install.py", line 69, in execute
    install(args, parser, 'install')
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\cli\install.py", line 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\plan.py", line 600, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\instructions.py", line 136, in execute_instructions
    cmd(state, arg)
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\instructions.py", line 59, in EXTRACT_CMD
    extract(arg)
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\install.py", line 852, in extract
    exp_backoff_fn(os.rename, temp_path, path)
  File "C:\Users\gthir\Anaconda2\lib\site-packages\conda\install.py", line 238, in exp_backoff_fn
    result = fn(*args)
WindowsError: [Error 5] Access is denied

[Anaconda2] C:> conda info
Current conda install:

```
         platform : win-64
    conda version : 4.1.5
conda-env version : 2.5.1
```

  conda-build version : 1.20.0
       python version : 2.7.12.final.0
     requests version : 2.9.1
     root environment : C:\Users\gthir\Anaconda2  (writable)
  default environment : C:\Users\gthir\Anaconda2
     envs directories : C:\Users\gthir\Anaconda2\envs
        package cache : C:\Users\gthir\Anaconda2\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
         offline mode : False
    is foreign system : False

**Comment by mcg1969** [comment]
Are you running a virus scanner? If so, can you temporarily disable it and try again? We've just identified a fixed for issues like this that are caused by virus scanner interventions.

**Comment by gthiruna** [comment]
Yeah Sure will try that out, Also we can install it using the pip command that also seems to work.
Is this a right way to fix the issue by installing it from pip instead of conda

**Comment by mcg1969** [comment]
No, I definitely recommend minimizing the use of pip in conda environments. That's not much consolation when experiencing a conda bug, no doubt. But if the virus scanner fix works for you then we can be confident this kind of problem won't happen for you again.

**Comment by gthiruna** [comment]
Sure Michael Will try it out after disabling the virus scanner and try to install it once again.
Thanks for the help and support mate :)

**Comment by kalefranz** [comment]
Permissions errors and multi-user/read-only installs have been fixed in later versions of conda 4.3.  Also fixed with transactions in 4.3 is conda removing itself during a failed install/update operation.
